app/models/public/study.rb

```diff
@@ -22,7 +22,6 @@ class Study < Public::PublicBase
     self.primary_key = 'nct_id'
 
     def self.ids_tagged(tag)
-      terms=Public::TaggedTerm.terms_for_tag(tag).pluck(:term).uniq
       ids=Public::BrowseCondition.ids_for_terms(terms) + Public::BrowseIntervention.ids_for_terms(terms)
```

Comment:
You can't just remove that line. This method is being given a tag, that tag is setting the term variable that is being called in this line `ids=Public::BrowseCondition.ids_for_terms(terms) + Public::BrowseIntervention.ids_for_terms(terms)`
This method will throw an error when called.

Check to see if this method is called anywhere. If it is not called then you can remove it. If it is called then we will need to discuss keeping the TaggedTerm model.